iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractIdentifiable.java

```diff
@@ -17,9 +17,9 @@
  */
 abstract class AbstractIdentifiable<I extends Identifiable<I>> extends AbstractExtendable<I> implements Identifiable<I>, Validable, MultiVariantObject {
 
-    protected String id;
+    protected final String id;
 
-    protected String name;
+    protected final String name;
```

Comment:
To be discussed: final vs setter

Reply:
To be checked: in iidm.xsd, if the network doesn't have a name attribute, add the setter in AbstractConnectable